pkg/catalog/endpoint_test.go

```diff
@@ -139,22 +140,20 @@ func TestListAllowedUpstreamEndpointsForService(t *testing.T) {
 
 			mockConfigurator := configurator.NewMockConfigurator(mockCtrl)
 			mockKubeController := k8s.NewMockController(mockCtrl)
-			mockEndpointProvider := endpoint.NewMockProvider(mockCtrl)
-			mockServiceProvider := service.NewMockProvider(mockCtrl)
+			mockProvider := compute.NewMockInterface(mockCtrl)
```

Comment:
nit: This applies to a couple of other files as well
```suggestion
			mockComputeInterface := compute.NewMockInterface(mockCtrl)
```